src/main/java/lotto/model/LottoTicketGenerator.java

```diff
@@ -0,0 +1,18 @@
+package lotto.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoTicketGenerator {
+
+    private static final List<LottoNumber> LOTTO_NUMBERS = IntStream.rangeClosed(LottoNumber.LOWER_BOUND, LottoNumber.UPPER_BOUND)
+            .mapToObj(LottoNumber::new)
+            .collect(Collectors.toList());
+
+    public static LottoTicket generate() {
+        Collections.shuffle(LOTTO_NUMBERS);
+        return new LottoTicket(LOTTO_NUMBERS.stream().limit(LottoTicket.SIZE).collect(Collectors.toList()));
```

Comment:
아래와 같이 `collectingAndThen`를 이용해보시면 조금 더 깔끔해질 것 같아요. 😃

```suggestion
        return LOTTO_NUMBERS.stream()
            .limit(LottoTicket.SIZE)
            .collect(Collectors.collectingAndThen(Collectors.toList(), LottoTicket::new));
```

Reply:
반영하였습니다!